react-rendering-lists/my-app/src/PokemonList.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = {
+  pokedex: {
+    name: string;
+    number: string;
+  }[];
```

Comment:
I would have created a sub-type, like:
```
type Pokedex = {
  number: string;
  name: string;
};
```
Then I would use it here and in line 9.